_src/pattern-library/sass/components/_grid.scss

```diff
@@ -0,0 +1,1125 @@
+// #UTILITY
+// #CANNED
+// #CANNED - RESPONSIVE
+// #GALLERY
+// #GALLERY - CANNED
+// #GALLERY - CANNED + RESPONSIVE
+// #DEBUG
+
+// ----------------------------
+
+// ----------------------------
+// #CONFIG
+// ----------------------------
+
+// susy-based configuration (see http://susydocs.oddbird.net/en/latest/settings/)
+$susy: (
+    use-custom: (
+        background-options: false,
+        box-sizing: false,
+        breakpoint: true,
+        clearfix: true,
+        rem: true
+    ),
+    math: fluid,
+    output: float
+);
+
+// LAYOUT: set default susy settings
+// (gutters are handled outside of elements - by default)
+$susy-default: (
+    column-width: $grid-column-width,
+    columns: $grid-column-count,
+    container-position: $grid-container-position,
+    container: $grid-max-width,
+    global-box-sizing: border-box,
+    gutters: $grid-gutter-width,
+    gutter-position: $grid-gutter-position
+);
+
+// LAYOUT: set susy settings especially for manually set grids in the DOM
+// (gutters are handled inside of elements - by default)
+$susy-manual: layout($susy-default $grid-manual-gutter-position);
+
+// LAYOUT: set debug settings
+$susy-debug: (
+    debug: (
+        color: $grid-debug-color,
+        image: show
+    )
+);
+
+
+// ----------------------------
+// #GLOBALS
+// ----------------------------
+// set basic grid settings for app
+@include layout($susy-default);
+
+
+// ----------------------------
+// #UTILITY
+// ----------------------------
+%grid-container {
+    @include container();
+}
+
+// grid rows - set visual break and document flow clear (clearfix)
+%grid-row {
+    @include break();
+}
+
+%grid-gallery {
+    @include break();
+}
+
+.grid-container {
+    @extend %grid-container;
+
+    .col.first {
+        @include first();
+    }
+
+    .col.last {
+        @include last();
+    }
+
+    .col.break {
+        @include break();
+    }
+
+    .col.omega {
+        @include omega();
+    }
+}
+
+.row {
+    @extend %grid-row;
+}
+
+// break grid's columns to a new line
+.grid-break {
+    @include break;
+}
+
+// don't break the columns to a new line
+.grid-unbreak {
+    @include nobreak;
+}
+
+
+// ----------------------------
+// #CANNED
+// ----------------------------
+.grid-manual {
+    @include layout($susy-manual, clean);
+
+    // column widths
+    .col-1 {
+        @include span(1);
+    }
+
+    .col-2 {
+        @include span(2);
+    }
+
+    .col-3 {
+        @include span(3);
+    }
+
+    .col-4 {
+        @include span(4);
+    }
+
+    .col-5 {
+        @include span(5);
+    }
+
+    .col-6 {
+        @include span(6);
+    }
+
+    .col-7 {
+        @include span(7);
+    }
+
+    .col-8 {
+        @include span(8);
+    }
+
+    .col-9 {
+        @include span(9);
+    }
+
+    .col-10 {
+        @include span(10);
+    }
+
+    .col-11 {
+        @include span(11);
+    }
+
+    .col-12 {
+        @include span(full);
+    }
+
+    // shifts - before cols
+    .pre-1 {
```

Comment:
NOTE: This could be made more efficient through a mixin or function, but that seemed like a clean up/optimization task once the grid gets some use.